cloudwatch_logger/include/cloudwatch_logger/log_node.h

```diff
@@ -33,15 +33,32 @@ namespace Utils {
 class LogNode : public Service
 {
 public:
+  struct Options {
```

Comment:
Better name? options is too vague.

Reply:
But it is nested under the `LogNode` class, so it will be named `LogNode::Options` for users outside of `LogNode`. That should be okay, right?

Reply:
Fair!